api/src/pcapi/alembic/versions/20240315T101954_a613c5bc707a_remove_feature_flage_enable_cgr_timeout.py

```diff
@@ -0,0 +1,30 @@
+"""Remove FF ENABLE_CGR_TIMEOUT"""
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "a613c5bc707a"
+down_revision = "4df45e259f14"
+branch_labels = None
+depends_on = None
+
+
+def get_flag():  # type:ignore[no-untyped-def]
```

Comment:
On ne type pas les fichiers dans `alembic/` (cf. [config mypy](https://github.com/pass-culture/pass-culture-main/blob/d5caf0edc681a3d846c1c1e11939982ee10a7f24/api/pyproject.toml#L152-L157)), ça évite d'encombrer notre code de `# type: ignore` (que l'on compte car on essaie de les éviter).

```suggestion
def get_flag():
```

Reply:
Sur cette ligne, j'ai une erreur avec pylint. Mais peut être que ça vient de mon environnement.